package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A work-in-progress bot used to link to Mojira bug reports directly from Discord.",
   "main": "index.js",
   "scripts": {
+    "lint": "eslint **/*.ts --fix",
```

Comment:
This command doesn't appear to work properly, presumably because this project uses TypeScript ESLint, not vanilla ESLint.

Reply:
But it works on my machine 🤔

Reply:
Also works for me

Reply:
Ah, I tried to run that command directly on the command line instead of using the `npm run lint` command, which obviously didn't work.